tests/console/force_cron_run.pm

```diff
@@ -30,18 +30,27 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # "bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with uptime less than 15 minutes
+    # because no daily jobs got created or can be excecuted.
+    # we stop and mask btrfs systemd timers. So keep bash -x /usr/lib/cron/run-crons for SLES 12
+    if (is_sle('<15') || is_leap('<15')) {
+        assert_script_run("bash -x /usr/lib/cron/run-crons", 1000);
+    }
+    else {
+        assert_script_run('rpm -q btrfsmaintenance && systemctl start btrfs-balance.service');
```

Comment:
I suggest to start *all* the timers to be more near to what we had for SLE12:

```
for i in $(sudo systemctl list-units --type=timer --state=active --no-legend -p=name | awk '{ print $1 }') ;do systemctl start $i ; done
```

there might be something nicer than awk but could not find a way to output only the name of the services easily. And we should not even need to check for the btrfsmaintenance package

@coolo this should ensure we do not need to hardcode service names and basically do the same as in before, run all timed jobs once. WDYT? Or should we rewrite the SLE12 case also to just point to specific services?

I don't think that btrfs-balance is enough because problems can be caused by the other btrfs maintenance jobs as well and also snapper(-cleanup), right?

Reply:
The 'sudo' is obviously not needed
'--property/-p' is only defined for the 'show' subcommand